pyproject.toml

```diff
@@ -9,6 +9,12 @@ license = "Apache-2.0"
 packages = [
   { include = "knockoff" },
 ]
+maintainers = [
+  "Gregory Yu <gregory.yu@nike.com>",
```

Comment:
Maintainers list should be distinct from Authors list
https://python-poetry.org/docs/pyproject/